tools/benchmark/cmd/mvcc-put.go

```diff
@@ -106,16 +108,25 @@ func mvccPutFunc(cmd *cobra.Command, args []string) {
 	rrc := r.Results()
 
 	rc := r.Run()
-	for i := 0; i < totalNrKeys; i++ {
-		st := time.Now()
-		if txn {
+
+	if txn {
+		for i := 0; i < totalNrKeys; i += nrBatch {
+			st := time.Now()
+
 			tw := s.Write()
-			tw.Put(keys[i], vals[i], lease.NoLease)
+			for j := i; j < i+nrBatch; j++ {
+				tw.Put(keys[j], vals[j], lease.NoLease)
+			}
 			tw.End()
-		} else {
+
+			rrc <- report.Result{Start: st, End: time.Now(), Ops: uint(nrBatch)}
```

Comment:
OK how about instead of `Ops`, there's `Weight float64` field? `Report` can compute the weighted rps / avg / stddev in addition to the usual stats.